### DISCUSSION TITLE
Cannot access manual Drop Down Menues - evtl Bug?

### DISCUSSION THREAD

**Discussion by Bloodiko** [netbox-community/netbox#7142] [Q&A]
Hello, 

i've upgraded from 2.11.7 to 3.0.0 and now to 3.0.1 

Now in 3.0.0 as well as 3.0.1 i've noticed something odd: 

I cannot access the Drop Down Menus, which are filled by Custom Input. 
For Example: 

DCIM->Sites 

Whenever I try to add a new Device or anything similiar, where I need to Access some Fields like Device-Type, Device-Role,...
Then the Drop-Down Menu is just empty. 

Same thing happens on "Filters" , when I want to Filter for a specific Device-Type. 

Did I miss a step on Upgrading ? 

![image](https://user-images.githubusercontent.com/21989414/131823767-1642dba3-dde8-4011-bada-f118c3011025.png)

Example Picture of an Empty Site drop down menu. 
I have several sites configured, and i can see as well as access them all in the menu. 

Thanks,

Ebiko

**Comment by candlerb** [comment]
It sounds like an Ajax problem.  Which browser are you using?

Firstly, try doing a shift-reload to force all Javascript to be reloaded.

If that doesn't fix it, can you open your browser developer console, and see if any Network errors are shown when you click on one of those fields. (e.g. in Chrome: three dots, More Tools, Developer Console).

**Reply by Bloodiko** [reply]
Okay, thats interesting. 
```
TypeError: e.replaceAll is not a function
    at xL (netbox.js:8)
    at netbox.js:8
    at Generator.next (<anonymous>)
    at netbox.js:1
    at new Promise (<anonymous>)
    at Ne (netbox.js:1)
    at pg (netbox.js:8)
    at netbox.js:9
    at Generator.next (<anonymous>)
    at netbox.js:1
(anonymous) @ netbox.js:9
a @ netbox.js:1
Promise.then (async)
c @ netbox.js:1
(anonymous) @ netbox.js:1
Ne @ netbox.js:1
loadData @ netbox.js:9
Cl.slim.beforeOpen @ netbox.js:9
f.open @ netbox.js:9
m.onclick @ netbox.js:9
```

It may be cause by an too old Chrome Version. (Chrome v78.  ) Since i'm in a gap Closed environment, im forced to use an older version. 
(The firefox version is even worse, the Left Sidebar is not useable at all in the Firefox Version 68.... ) 

Maybe im really forced to use Edge, unless i get a newer Chrome version.
